app/code/pagetype/HomePage.php

```diff
@@ -61,20 +82,89 @@ public function getCMSFields()
         $fields = parent::getCMSFields();
 
         // Banner Tab
-        $fields->addFieldToTab('Root.Banner', TextField::create(
-            'BannerHeading',
-            'Banner Heading'
-        ));
-
-        $fields->addFieldToTab('Root.Banner', TextField::create(
-            'BannerDesc',
-            'Banner Description'
-        ));
-
-        $fields->addFieldToTab('Root.Banner', $bannerImage = UploadField::create(
-            'BannerImage',
-            'Banner Image'
-        )->setDescription('Only supports <strong>jpg, jpeg, png</strong> filetypes.</br>Recommended dimensions 1920 x 1080 px.'));
+        $fields->addFieldToTab(
```

Comment:
A nice refactor we can do here is using the plural form of `addFieldToTab` which is `addFieldsToTab`.

We can take all the fields related to `Root.Banner` and put it into an array like so:
```
$fields->addFieldsToTab(
     'Root.Banner'
     [
          TextField...,
          TextField...,
          UploadField...,
     ]
);
 
This way we're not making multiple function calls for the same tab :)